src/services/auth.ts

```diff
@@ -165,6 +177,13 @@ class AuthService {
    * where we could get them directly via User.profile.
    */
   async getUser(oidcUser?: OidcUser | null): Promise<User | null> {
+    let user: User | null = null;
+
+    let userLocalStore = localStorage.getItem("userObj")
+    let userLocalStoreJSON: any;
+    userLocalStore && userLocalStore !== "null" ? userLocalStoreJSON = JSON.parse(userLocalStore) : user = null
+    userLocalStoreJSON ? user = { email: userLocalStoreJSON.email, extId: userLocalStoreJSON.extId, expired: userLocalStoreJSON.expired, name: userLocalStoreJSON.name, fullName: userLocalStoreJSON.fullName, loginState: userLocalStoreJSON.loginState } : user = null
+
```

Comment:
These 5 lines above should go into the else-branch of the `if (oidcuser)` below.

And again, not localStore but sessionStorage should be used.

Maye you should catch the case that the JSON.parse throws an error, though it *should* never happen.

Also, I found these lines a bit hard to read. Probably you're overusing the assignment operator.

I think `if (p) a = x; else b = y;` is better than `p ? a = x : b = y` because I expect the expressions in the conditional operator to be pure expressions, not statements with side effects (like assignments).

In the case of `p ? a = x : a = y` I would agree that a conditional operator is useful, but it should be written as  `a = p ? x : y`, not `p ? a = x: a = y`.

Similar reason why I would use a standard for loop in Python to *do* something, and a list comprehension to just create a list, without side effects, even though you could abuse it to have side effects.